assets/src/application/ui/layout/entityList/pagination/style.scss

```diff
@@ -8,6 +8,12 @@
 	flex-flow: row nowrap;
 	justify-content: flex-end;
 
+	.ant-pagination-options-size-changer {
+		&.ant-select {
+			width: 105px;
+		}
```

Comment:
maybe use `rem` for units which will scale better if font sizes are adjusted

Reply:
used rems, thx